fields/_config.php

```diff
@@ -1,32 +1,42 @@
-<?php
-
-/*
- * Custom Fields Config File
- * Description: This is custom fields config file. Require your custom field's "main" file here.
+<?php // @codingStandardsIgnoreLine.
```

Comment:
is @codingStandardsIgnoreLine required here?

Reply:
Yes.

FILE: C:\xampp\htdocs\lite-testing\wp-content\plugins\ultimate-addons-for-beaver-builder-lite\fields\_config.php                                                                                                                  -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------                                                   FOUND 2 ERRORS AFFECTING 1 LINE                                                                                                                                                                                                   -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------                                                    1 \| ERROR \| Filenames should be all lowercase with hyphens as word separators. Expected -config.php, but found _config.php.                                                                                                         \|       \| (WordPress.Files.FileName.NotHyphenatedLowercase)
--